test/SourceKit/CursorInfo/cursor_info.swift

```diff
@@ -422,31 +422,29 @@ func convention7(_: @convention(witness_method) ()->()) {}
 // CHECK33-NEXT: s:V11cursor_info2S2
 // CHECK33-NEXT: S2<T, U>.Type
 // CHECK33: <Declaration>struct S2&lt;T, U&gt; where U == T</Declaration>
-// CHECK33-NEXT: <decl.struct><syntaxtype.keyword>struct</syntaxtype.keyword> <decl.name>S2</decl.name>&lt;<decl.generic_type_param usr="s:tV11cursor_info2S21TMx"><decl.generic_type_param.name>T</decl.generic_type_param.name></decl.generic_type_param>, <decl.generic_type_param usr="s:tV11cursor_info2S21UMq_"><decl.generic_type_param.name>U</decl.generic_type_param.name></decl.generic_type_param>&gt; <syntaxtype.keyword>where</syntaxtype.keyword> U == T</decl.struct>
+// CHECK33: <Declaration>struct S2&lt;T, U&gt; where T == U</Declaration>
+// CHECK33-NEXT: <decl.struct><syntaxtype.keyword>struct</syntaxtype.keyword> <decl.name>S2</decl.name>&lt;<decl.generic_type_param usr="s:tV11cursor_info2S21TMx"><decl.generic_type_param.name>T</decl.generic_type_param.name></decl.generic_type_param>, <decl.generic_type_param usr="s:tV11cursor_info2S21UMq_"><decl.generic_type_param.name>U</decl.generic_type_param.name></decl.generic_type_param>&gt; <syntaxtype.keyword>where</syntaxtype.keyword> <decl.generic_type_requirement>T == U</decl.generic_type_requirement></decl.struct>
 
 // RUN: %sourcekitd-test -req=cursor -pos=81:8 %s -- -F %S/../Inputs/libIDE-mock-sdk -I %t.tmp %mcp_opt %s | %FileCheck %s -check-prefix=CHECK34
 // CHECK34: source.lang.swift.decl.function.method.instance (81:8-81:50)
 // CHECK34-NEXT: foo(_:)
-// CHECK34-NEXT: s:FV11cursor_info2S23foou0_Rd_0_zqd__rFFT_T_FT_T_
-// CHECK34-NEXT: <T, U, V, W where U == T, W == V> (S2<T, U>) -> (() -> ()) -> () -> ()
```

Comment:
Ah, this was a conflict resolution when I cherry-picked the original commit. I'll fix it.